### PR TITLE
Extend postgis ignore list to views on postgres

### PR DESCRIPTION
**Pull request by tomhoule**
spatial_ref_sys and geometry_columns used to be tables managed by
postgis, and in more recent versions they are views managed by postgis.
We make sure to ignore them in both cases.

closes https://github.com/prisma/prisma/issues/7764

### CODE REVIEW

File: migration-engine/connectors/sql-migration-connector/src/lib.rs [hunk 1]
```diff
@@ -130,6 +130,7 @@ impl SqlMigrationConnector {
         // accidentally drop something we can't describe in the data model.
         let drop_views = source_schema
             .view_walkers()
+            .filter(|view| !self.flavour.view_should_be_ignored(view.name()))
```
Comment (tomhoule) [inline migration-engine/connectors/sql-migration-connector/src/lib.rs:133]: That would be testable. i'll add a test.

Comment (tomhoule) [reply migration-engine/connectors/sql-migration-connector/src/lib.rs:133]: Done

**Comment by tomhoule** [comment]
Second commit is completely unrelated. You can review the first and ignore the second.

File: migration-engine/migration-engine-tests/src/commands/schema_push.rs [hunk 2]
```diff
@@ -11,27 +11,17 @@ pub struct SchemaPush<'a> {
     force: bool,
     /// Purely for logging diagnostics.
     migration_id: Option<&'a str>,
-    rt: Option<&'a tokio::runtime::Runtime>,
+    rt: &'a tokio::runtime::Runtime,
 }
 
 impl<'a> SchemaPush<'a> {
-    pub fn new(api: &'a dyn GenericApi, schema: String) -> Self {
+    pub fn new(api: &'a dyn GenericApi, schema: String, rt: &'a tokio::runtime::Runtime) -> Self {
```
Comment (pimeys) [inline migration-engine/migration-engine-tests/src/commands/schema_push.rs:18]: sneaky

**Review by pimeys — APPROVED** [review]
🥇

File: migration-engine/migration-engine-tests/src/commands/schema_push.rs (hunk 2 above)
Comment (tomhoule) [reply migration-engine/migration-engine-tests/src/commands/schema_push.rs:18]: Just deleted `new()` ande renamed `new_sync()` to `new()`.

**Pull request merged**